app/models.py

```diff
@@ -888,14 +888,14 @@ def apply(self, domain, post_records):
 
         # Adjustment to add multiple records which described in https://github.com/ngoduykhanh/PowerDNS-Admin/issues/5#issuecomment-181637576
         final_records = []
-        if NEW_SCHEMA:
-            records = sorted(records, key = lambda item: (item["name"], item["type"]))
-            for key, group in itertools.groupby(records, lambda item: (item["name"], item["type"])):
+        records = sorted(records, key = lambda item: (item["name"], item["type"]))
+        for key, group in itertools.groupby(records, lambda item: (item["name"], item["type"], item["changetype"], item["ttl"])):
+            if NEW_SCHEMA:
                 new_record = {
                         "name": key[0],
                         "type": key[1],
-                        "ttl": records[0]['ttl'],
```

Comment:
This sets the TTL to the same for all entries in the RRset, using the first TTL-value in the records-list.